runtime/doc/lsp.txt

```diff
@@ -425,6 +425,14 @@ therefore the linked highlights), use the |:highlight| command: >
     highlight LspDiagnosticsDefaultError guifg="BrightRed"
 <
 
+Note: Most custom colorschemes call `highlight clear` by convention on
+initialization. If the colorscheme does not define highlight groups for
+LSP diagnostics, you can set the highlights after the colorscheme has
+initialized with an autocommand. You will need to set this autocommand
+before enabling the colorscheme: >
```

Comment:
If a blank line follows anyway, the `>` could be moved to the start of the next line. Just saying, no need to change it again. :)

The wording is clearer now.